lua/astrocommunity/pack/godot/init.lua

```diff
@@ -20,4 +57,39 @@ return {
       end
     end,
   },
+  {
+    "nvim-neo-tree/neo-tree.nvim",
+    opts = function(_, opts)
+      -- Add renaming Godot UID files with file renames
+      opts.event_handlers = opts.event_handlers or {}
+      table.insert(opts.event_handlers, {
+        event = "file_renamed",
+        handler = function(args) move_uid_handler(args.source, args.destination) end,
+      })
+      table.insert(opts.event_handlers, {
+        event = "file_moved",
+        handler = function(args) move_uid_handler(args.source, args.destination) end,
+      })
+
+      -- Add Godot UID files to the filter list
+      if vim.g.godot_hide_uid ~= false then
+        local never_show = vim.g.godot_hide_uid == "never"
+
+        opts.filesystem = opts.filesystem or {}
+        opts.filesystem.filtered_items = opts.filesystem.filtered_items or {}
+
+        if never_show then
+          opts.filesystem.filtered_items.never_show_by_pattern = opts.filesystem.filtered_items.never_show_by_pattern
+            or {}
+          table.insert(opts.filesystem.filtered_items.never_show_by_pattern, "*.uid")
+        else
+          opts.filesystem.filtered_items.hide_by_pattern = opts.filesystem.filtered_items.hide_by_pattern or {}
+          table.insert(opts.filesystem.filtered_items.hide_by_pattern, "*.uid")
+        end
+      end
+    end,
+  },
+  {
+    "Cretezy/godot-server.nvim",
```

Comment:
```suggestion
    "Cretezy/godot-server.nvim",
    opts = {}, 
```

Reply:
It doesn't need this, it's marked as `lazy = false` inside the plugin itself